CMakeLists.txt

```diff
@@ -15,6 +14,10 @@ include(${ROOT_DIR}/cmake/common.cmake NO_POLICY_SCOPE)
 # internal cmake settings
 #------------------------------------------------------------------------------
 
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
It seems conceivable to me that we inherit bits of code that need to be compiled as C or not as C++11.  In changes I'm making for #1922, I stuck this in the function pdal_target_compile_settings().  Is that better than global? Just wondering.

Reply:
I'll take it out here, and let you define it on the targets where we know we want it.

Reply:
I don't really know how these play together. I wonder if it is still worthwhile to disable extensions, even while leaving the standard undefined. Maybe that's a per target thing as well.